cloud-optimized-geotiffs/cogs-examples.ipynb

```diff
@@ -9,6 +9,7 @@
    ]
```

Comment:
<p>we should cross link this section the the new noteboook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
fixed!